addons/website/tests/test_page.py

```diff
@@ -283,7 +283,7 @@ def test_03_error_page_debug(self):
             self.assertIn('ZeroDivisionError: division by zero', r.text, "Error should be shown in debug.")
 
     def test_04_visitor_no_session(self):
-        with patch.object(root.session_store, 'save') as session_save,\
+        with patch.object(root.session_store, 'save') as session_save, \
```

Comment:
On the python-side, this is the only issue fixed in this PR that I could not get triggered by a linter rule. Do you know which rule was infringed on your side ?

Reply:
![image](https://github.com/odoo/odoo/assets/30048408/a3f9eb0c-3799-4409-84a6-8e5b8429661d)
E231 it seems